apps/meteor/app/livechat/server/lib/sendTranscript.ts

```diff
@@ -52,15 +53,6 @@ export async function sendTranscript({
 	const timezone = getTimezone(user);
 	logger.debug(`Transcript will be sent using ${timezone} as timezone`);
 
-	if (!room) {
-		throw new Error('error-invalid-room');
-	}
-
-	// allow to only user to send transcripts from their own chats
-	if (room.t !== 'l') {
```

Comment:
no need to check this anymore?

Reply:
Should come for free on the `LivechatRooms.findOne` check (as it has the t: 'l' filter inside)